### ISSUE TITLE
Add message arguments to `declaration-property-value-disallowed-list`

### ISSUE THREAD

**Issue by glen-84** [stylelint/stylelint#6453]
### What is the problem you're trying to solve?

#4117 has been closed without a solution for `declaration-property-value-disallowed-list`. I'd still like to be able to specify a custom message per disallowed property value.

### What solution would you like to see?

```js
"declaration-property-value-disallowed-list": [
    {
        "line-height": ["normal"]
    },
    {
        message: (propertyName, _propertyValues) => {
            switch (propertyName) {
                case "line-height":
                    return "Avoid setting line-height to 'normal', as the value depends on the user agent";
            }
        }
    }
]
```

**Comment by ybiquitous** [comment]
@glen-84 Thanks for the proposal. In PR #6312, we've started supporting just-needed rules since we were not confident in the feature.

So, if there are needs, we can easily increase supported rules by passing `messageArgs` to `report()` like this (see also #6391):

https://github.com/stylelint/stylelint/blob/2724aa4f56113203328a438ca6e35f186253e2c6/lib/rules/color-no-hex/index.js#L44-L46

---

I've labeled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.
